hikari/users.py

```diff
@@ -180,6 +207,53 @@ def mention(self) -> str:
             The mention string to use.
         """
 
+    def make_banner_url(self, *, ext: typing.Optional[str] = None, size: int = 4096) -> typing.Optional[files.URL]:
+        """Generate the banner URL for this user, if set.
+
+        If no custom banner is set, this returns `builtins.None`.
+
+        Parameters
```

Comment:
This is inconsistent with how methods like make_avatar_url are declared on User since PartialUser can't guarantee that it'll know whether they have a banner or not